### PR TITLE
server plugins: convolution ugens: initialize outputs

### PR DESCRIPTION
**Pull request by snappizz**

### CODE REVIEW

File: server/plugins/Convolution.cpp [hunk 1]
```diff
@@ -766,6 +775,10 @@ void StereoConvolution2L_Ctor(StereoConvolution2L *unit)
 		unit->m_prevtrig = 0.f;
 
 		SETCALC(StereoConvolution2L_next);
+
+		// initialize outputs
+		OUT0(0) = IN0(0);
+		OUT0(1) = IN0(1);
```
Comment (Sciss) [inline server/plugins/Convolution.cpp]: I don't understand this line, but _warning_ I have no big clue about how the UGen code works. I'm just thinking that `OUT0(1)` might be the second sample of the first (left) output and not (as intended?) the first sample of the second (right) output. Also, the input to `StereoConvolution` is monophonic, so `IN0(1)` would probably also not be correct?

Comment (snappizz) [inline server/plugins/Convolution.cpp]: `OUT0(1)` is definitely correct (see the recently written Server Plugin API page), but `IN0(1)` is not. Thanks for catching.

**Comment by snappizz** [comment]
Here's a test case for Convolution (I'll edit this comment to add test cases for others):

```
s.record;
{ Delay1.ar(Convolution.ar(SinOsc.ar(440), SinOsc.ar(1000), 1024)); }.plot(0.05);
```

**Comment by snappizz** [comment]
This is the last remaining PR for #2333.

**Comment by vivid-synth** [comment]
Thanks @snappizz ! Our un-shredded ears thank you :smile:

**Pull request merged**